File: Bai_72/main.cpp
#include<iostream>
using namespace std;
int main(){
    char d;
    do
    {   
        float a, b;
        cout << "a="; cin >> a;
        cout << "b="; cin >> b;
        char c;
        cout << "Toan tu:"; cin >> c;
        switch (c){
        case '+': cout << a << c << d << "=" << a + b; break;
        case '-': cout << a << c << d << "=" << a - b; break;
        case '*': cout << a << c << d << "=" << a * b; break;
        case '/': cout << a << c << d << "=" << a / b; break;
        }
        cout << "\nTiep tuc:"; cin >> d;
    } while (d != 't' && d != 'T');
    return 0;
}

File: Bai_72/main.py
while True:
    a = float(input("a="))
    b = float(input("b="))
    c = input("Toan tu:")
    if c == '+':
        print(a, c, b,"=",a+b)
    elif c == '-':
        print(a, c, b,"=",a-b)
    elif c == '*':
        print(a, c, b,"=",a*b)
    elif c == '/':
        print(a, c, b,"=",a/b)
    d = input("Tiep tuc:")
    if d == 't' or d == "T":
        break